### DISCUSSION TITLE
Very subtle focus highlighting on project nav bar buttons

### DISCUSSION THREAD

**Discussion by sbland** [community/community#14467] [Accessibility]
On a project page the focus highlight for the nav bar buttons (Code|Issues|Pull requests...) is very difficult to see. At first I didn't think there was any focus highlighting. Is it possible to increase the contrast on this?

Platform Windows
Using Chrome Version 100.0.4896.75 (Official Build) (64-bit)

**Comment by cehfisher** [comment]
Hello @sbland - thank you for providing this feedback! We wanted to follow up on this issue. Have the [two new high contrast themes](https://github.com/settings/appearance) resolved this issue for you? If not, could you provide more information on what changes you need and include screenshots and/or URLs, if applicable. Thank you!

**Reply by sbland** [reply]
Much better thanks!
